desktop/cmp/error/ErrorMessage.js

```diff
@@ -57,22 +78,21 @@ const titleCmp = hoistCmp.factory(
     }
 );
 
-const errorCmp = hoistCmp.factory(
-    ({error}) => {
-        if (error instanceof Error || error?.message) return p(error.message);
-        if (isValidElement(error)) return error;
-        if (isString(error)) return p(error);
+const messageCmp = hoistCmp.factory(
+    ({message}) => {
+        if (isValidElement(message)) return message;
+        if (isString(message)) return p(message);
         return null;
     }
 );
 
 const actionButton = hoistCmp.factory(
-    ({actionFn, actionButtonProps}) => {
+    ({actionFn, actionButtonProps, error}) => {
         if (!isFunction(actionFn)) return null;
         return button({
             text: 'Retry',
             minimal: false,
-            onClick: () => actionFn(),
+            onClick: () => actionFn({error}),
```

Comment:
Should the argument here just be `error` (not an object)?

I can see the object being useful if we think we're going to extend this, but not sure what else we would pass. As it stands, I think an object with one property is more unexpected than just passing the error.